data/mods/Xedra_Evolved/items/inventor/artifacts.json

```diff
@@ -102,8 +102,8 @@
     "type": "TOOL",
     "id": "mace_magic",
     "name": { "str": "wolf sledge" },
-    "description": "A long, solid, crafted steel haft, and a massive, comma-like head.  It feels really massive, but it is easy to wield.  You even can whisper ancient words to it while you throw it, and it will always return to your hands.",
-    "weight": "2950 g",
+    "description": "A long, solid, crafted steel haft, and a massive, comma-like head.  It feels really massive, but it is easy to wield, and each your level it deal more and more damage.  You even can whisper ancient words to it while you throw it, and it will always return to your hands.",
```

Comment:
```suggestion
    "description": "A long, solid, crafted steel haft, and a bulky, comma-like head.  It feels really massive, but it is surprisingly easy to wield, and you will do more damage as your level increases.  You can make it return by speaking its command word, which happens to be in an ancient, long forgotten language, while throwing.",
```

Reply:
why it must be `in an ancient, long forgotten language`?

Reply:
i dunno. i just reworded it so it sounds better. if it doesnt fit, remove the long forgotten part.